providers/implementations/rands/drbg_ctr.c.in

```diff
@@ -338,13 +344,18 @@ static int drbg_ctr_instantiate_wrapper(void *vdrbg, unsigned int strength,
                                         const OSSL_PARAM params[])
 {
     PROV_DRBG *drbg = (PROV_DRBG *)vdrbg;
+    struct drbg_set_ctx_params_st p;
     int ret = 0;
 
+    if (drbg == NULL || !drbg_ctr_set_ctx_params_decoder(params, &p))
+        return 0;
+
+
```

Comment:
Nit: extra new line

Reply:
Fixed.  I don't such a trivial change this should invalidate the reviews so I'll leave the approval done in place.